pointcloud/src/main/scala/geotrellis/pointcloud/raster/ept/EPTTreeMetadata.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.pointcloud.raster.ept
+
+import geotrellis.proj4._
+import geotrellis.raster._
+import geotrellis.util.RangeReader
+
+import _root_.io.circe.parser._
+import cats.instances.long._
+import cats.instances.map._
+import cats.syntax.semigroup._
+
+import java.net.URI
+
+case class EPTTreeMetadata(
```

Comment:
I didn't remove the old metadata parser, just renamed it in case we'll still need it.